build/configs/sidk_s5jt200/scripts/ld_s5jt200_flash.script

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2016 Samsung Electronics All Rights Reserved.
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
```

Comment:
This file was pushed at 2016 first, not 2017. Don't modify this.

Reply:
will fix it.